ecs-init/apparmor/apparmor_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
super nit & non-blocking: maybe we should remove `2015-2018` from the copyright header since this is a new file. Ref: [here](https://github.com/aws/amazon-ecs-agent/blob/b6866da0ebc66b0024f88db043ac3cfb6cb2d901/agent/eventhandler/attachment_handler.go#L1-L12)!

Reply:
done